lib/js/components/Cards/Card/Card.vue

```diff
@@ -190,6 +119,25 @@
 			}
 		}
 	}
+
+	// the border container is added to avoid adding overflow hidden to the card itself
+	// which would clip floating elements rendered within the card (e.g. dropdowns)
+	&__border {
+		display: flex;
+		flex-shrink: 0;
+	}
+
+	&:not(.-ds-flat) &__border {
+		border-top-left-radius: $radius-m;
+		border-top-right-radius: $radius-m;
+		overflow: hidden;
+	}
+
+	&.-ds-leftBorder &__border {
+		border-bottom-left-radius: $radius-m;
+		border-top-left-radius: $radius-m;
```

Comment:
it doesn't change anything visually, but because of missing `&:not(.-ds-flat)` border radius is applied on flat card. It isn't visible because there is no `overflow: hidden`. For me it'd be safer if we limit this selector with `&:not(.-ds-flat)`